servarr/values.yaml

```diff
@@ -34,6 +34,49 @@ metrics:
   # @section -- Metrics
   enabled: &metricsEnabled false
 
+scraparr: 
+  enabled: false
+  config:  
+    radarr:
+      # -- (string) Mandatory - Insert Radarr service name in format: {{ .Release.Namespace }}-radarr
+      # @section -- Scraparr
+      # @default -- No default value  
+      url: 
+      # -- (string)  - Insert Radarr API versions, if different version wants to be used
+      # @section -- Scraparr
+      # @default -- v3
+      api_version: v3
+      # -- Anchor to apiKey. As a single key is used for all services, same will be used for interacting with Sonarr, Radarr, Prowlarr
+      # @default 
+      # @section -- scarparr 
+      api_key: *apikey
+    sonarr:
+      # -- (string) Mandatory - Insert Sonarr service name in format: {{ .Release.Namespace }}-sonarr
+      # @section -- Scraparr
+      # @default -- No default value  
+      url: 
+      # -- (string)  - Insert Sonarr API versions, if different version wants to be used
+      # @section -- Scraparr
+      # @default -- v3
+      api_version: v3
+      # -- Anchor to apiKey. As a single key is used for all services, same will be used for interacting with Sonarr, Radarr, Prowlarr
+      # @default 
+      # @section -- scarparr 
+      api_key: *apikey    
+    prowlarr:
+      # -- (string) Mandatory - Insert Prowlarr service name in format: {{ .Release.Namespace }}-prowlarr
+      # @section -- Scraparr
+      # @default -- No default value  
+      url: 
+      # -- (string)  - Insert Prowlarr API versions, if different version wants to be used
+      # @section -- Scraparr
+      # @default -- v3
+      api_version: v3
+    # -- Anchor to apiKey. As a single key is used for all services, same will be used for interacting with Sonarr, Radarr, Prowlarr
+    # @default 
+    # @section -- scarparr 
+    api_key: *apikey    
```

Comment:
This parameter here should be nested under `prowlarr` object, cause Scraparr will look for `scraparr.config.<sonarr|radarr|prowlarr>[i].api_key`